src/cmds/build/index.ts

```diff
@@ -1,17 +1,15 @@
-const builders = require('../../builders');
-const {
-  getOutfilePath,
-  validateDirPath,
-  validateFilePath,
-  validateOutfileName,
-} = require('../../utils');
-const { handler: snapEval } = require('../eval');
-const { handler: manifest } = require('../manifest');
-const { bundle } = require('./bundle');
+import builders from '../../builders';
+import { getOutfilePath, validateDirPath, validateFilePath, validateOutfileName } from '../../utils';
+import { snapEval } from '../eval';
+import { manifest } from '../manifest/manifest';
+import { bundle } from './bundle';
+
+/* Custom Type Imports */
+import { Argument } from "../../types/yargs";
 
 module.exports.command = ['build', 'b'];
 module.exports.desc = 'Build Snap from source';
-module.exports.builder = (yarg) => {
+module.exports.builder = (yarg: any) => {
```

Comment:
Can you expand on why we have to use `any` for the `yarg` parameter here and elsewhere? Is there no `@types/yargs` type that works?